code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -393,6 +393,19 @@
 /obj/machinery/smartfridge/chemistry
 	name = "smart chemical storage"
 	desc = "A refrigerated storage unit for medicine storage."
+	var/list/chemfridge_typecache
```

Comment:
can we just put this as a static list in accept_check?

Reply:
Anything for you Cobby